code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -254,20 +254,23 @@
 	icon_state = "cryog"
 	affected_area = 2
 	ignition_temp = -100
+	banned_containers =  list(/obj/item/reagent_containers/glass/beaker/beaker_of_holding)
 
 /obj/item/grenade/chem_grenade/pyro // Intended for pyrotechnical mixes. Produces a small fire upon detonation, igniting potentially flammable mixtures.
 	name = "pyro grenade"
 	desc = "A custom made pyrotechnical grenade. Heats up contents upon ignition."
 	casedesc = "Upon ignition, it rapidly heats contents by 500°K."
 	icon_state = "pyrog"
 	ignition_temp = 500 // This is enough to expose a hotspot.
+	banned_containers =  list(/obj/item/reagent_containers/glass/beaker/beaker_of_holding)
 
 /obj/item/grenade/chem_grenade/adv_release // Intended for weaker, but longer lasting effects. Could have some interesting uses.
 	name = "advanced release grenade"
 	desc = "A custom made advanced release grenade. It is able to be detonated more than once. Can be configured using a multitool."
 	casedesc = "This casing is able to detonate more than once. Can be configured using a multitool."
 	icon_state = "timeg"
 	var/unit_spread = 10 // Amount of units per repeat. Can be altered with a multitool.
+	banned_containers =  list(/obj/item/reagent_containers/glass/beaker/beaker_of_holding)

```

Comment:
dont

Reply:
Im not sure what you mean here. Do you mean this is not how it should be written or that the beaking of holding shouldnt be banned from the nades?

Reply:
you are overriding the parents banned containers to just ban beakers of holding, so now all grenades except basic ones would be able to fit in bluespace beakers, making large grenades useless

Reply:
ill add the bluespaces to the list

Reply:
actually no ill just remove the list entirely and let parent decide